test/test_functional_tensor.py

```diff
@@ -1,34 +1,58 @@
 from __future__ import division
 import torch
+from torch import Tensor
 import torchvision.transforms as transforms
 import torchvision.transforms.functional_tensor as F_t
 import torchvision.transforms.functional as F
 import numpy as np
 import unittest
 import random
+from torch.jit.annotations import Optional, List, BroadcastingList2
+from typing import Tuple
```

Comment:
Can we import `Tuple` from `torch.jit.annotations`, just to make things independent of typing for now?